bazel/foreign_cc/vpp_vcl.patch

```diff
@@ -80,75 +78,81 @@ index 04ebed548..bfabc3a67 100644
  install(
    FILES vppapigen.py
    RENAME vppapigen
-diff --git a/src/tools/vppapigen/vppapigen.py b/src/tools/vppapigen/vppapigen.py
-index 2b0ce9999..f8a7586ea 100755
---- a/src/tools/vppapigen/vppapigen.py
-+++ b/src/tools/vppapigen/vppapigen.py
-@@ -7,6 +7,14 @@ import logging
+diff --git src/tools/vppapigen/vppapigen.py src/tools/vppapigen/vppapigen.py
+index 0bed578eb..fb476f2bb 100755
+--- src/tools/vppapigen/vppapigen.py
++++ src/tools/vppapigen/vppapigen.py
+@@ -7,6 +7,13 @@ import logging
  import binascii
  import os
  from subprocess import Popen, PIPE
 +
-+
 +# Put ply on the path ...
 +plypath = os.path.join(
 +    os.environ["EXT_BUILD_ROOT"],
-+    os.path.dirname(os.path.dirname(os.environ["PLYPATHS"].split()[0])))
++    os.path.dirname(os.environ["PLYPATHS"].split()[0]))
```

Comment:
i think this is change is wrong - i just updated this to make rules_python update work - its a kinda hacky/brittle workaround but works with the extra level of `dirname`